### DISCUSSION TITLE
How to move secondary side bar to bottom?

### DISCUSSION THREAD

**Discussion by EKtheSage** [posit-dev/positron#3989] [Q&A]
Is there a way to arrange the layout to look like RStudio as seen here?

I like to keep Console and Editor side by side and plots/environment in the lower right corner.  But in positron I can only move the secondary sidebar to left.

RStudio:
![image](https://github.com/user-attachments/assets/b14f0c03-f18d-466e-800d-e95f16c171d5)

positron:
![image](https://github.com/user-attachments/assets/a3ed7c5d-9c88-4d8c-af1c-d939beabf0f1)

**Comment by juliasilge** [comment]
Try out our "side-by-side" layout (you can get to this from the layout button at the very top right):

<img width="631" alt="Screenshot 2024-07-12 at 9 26 07 AM" src="https://github.com/user-attachments/assets/834d2fe5-1c7f-4e5f-aafa-84e47144b7c6">

It looks like this:

<img width="1175" alt="side-by-side" src="https://github.com/user-attachments/assets/15ad08ed-515d-47b2-be9b-30b8c54c4559">

You can also toggle or drag around these panels to customize your layout further.

**Reply by EKtheSage** [reply]
Thanks @juliasilge , this almost fits my needs, but the plot pane probably needs to be optimized when resizing the pane. The example below uses `dygraph`, I haven't tried static plots like ggplot yet.

Panel showing the plot pane:
![image](https://github.com/user-attachments/assets/58ef07f0-27be-48e8-b636-a9ccc25bb503)

Panel minimizing the plot pane:
![image](https://github.com/user-attachments/assets/3c3df723-3400-4b6a-8271-5c9f21c563f5)

**Reply by juliasilge** [reply]
I believe this is the same problem as #2170, if you would like to add any additional context there!
